### PR TITLE
Add odometer.js for animation of top line number.

### PR DESCRIPTION
**Pull request by audiodude**
In regards to #59.

Not sure how this will work if there is like 1.2M displayed, or if that's even a thing.

### CODE REVIEW

**Comment by audiodude** [comment]
There was an odd typo that I fixed. I also reduced the duration. PTAL.

File: js/index.js [hunk 1]
```diff
@@ -78,7 +85,7 @@
     "realtime": renderBlock()
       .render(function(selection, data) {
         var totals = data.data[0];
-        selection.text(formatCommas(+totals.active_visitors));
```
Comment (audiodude) [inline js/index.js]: odometer.js does its own formatting

**Comment by konklone** [comment]
I'm again summoning @shawnbot @cew821 @rypan for this one. My gut reaction on seeing it as that it's not as subtle or tasteful as I imagined it would be in my head when we first discussed it.

**Comment by konklone** [comment]
The sense of the team was that this wasn't the direction we wanted to go. Thanks for submitting it, @audiodude!

**Pull request closed without merging**